11ty/index.njk

```diff
@@ -16,9 +16,17 @@ layout: layouts/index.njk
       <div class="box">
         <p class="subtitle">Ways to help</p>
         <ol class="help">
-          <li> Submit words and definitions through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a>.</li>
+          <li>Submit words and definitions through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a> or <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferral">issues</a>.</li>
           <li>Sponsor this work through <a href="https://github.com/sponsors/tatianamac">GitHub Sponsors</a>.</li>
```

Comment:
Maybe we can break it out as:

>Submit words and definitions or contribute to our code base. Start with our [contributing guidelines](https://github.com/tatianamac/selfdefined/blob/master/CONTRIBUTING.md) and [documentation](https://www.selfdefined.app/documentation/).
>Sponsor this work through [Open Collective](https://opencollective.com/selfdefined) or [GitHub sponsors](https://github.com/sponsors/tatianamac).
>Follow us on Twitter [@SelfDefinedApp](https://www.twitter.com/SelfDefinedApp).

Reply:
Does it mean you want to remove links to pull requests and issues?

Reply:
No, I think we can leave it. I was toying with the idea of forcing people to go to the documentation first but I think it's fine as is.